terraform/modules/critical/vhs_manifests.tf

```diff
@@ -1,4 +1,4 @@
 module "vhs_manifests" {
-  source = "git::https://github.com/wellcometrust/terraform-modules.git//vhs/modules/vhs?ref=v19.13.6"
+  source = "git::https://github.com/wellcometrust/terraform-modules.git//vhs/modules/vhs?ref=4660e69"
```

Comment:
this probably needs to be a fixed version -  we can do that in a future PR though